src/slider/index.html

```diff
@@ -32,6 +32,34 @@
 </head>
 <body>
 	<main>
+		<!--Slider with 'fade in' effect-->
+		<tp-slider flexible-height="yes" infinite="yes" swipe="yes" behaviour="fadein">
```

Comment:
@imranhsayed Lets rename `fadein` to just `fade` everywhere, and move this slider under the other one - as we want to feature the slider as the main feature